test/runify.js

```diff
@@ -442,8 +442,56 @@ hosts.forEach(function (record) {
         }).then(record => {
           const result = record[0];
           assert(!result.stdout.match(/2/), 'Unexpected stdout: ' + result.stdout);
-        })
-      })
+        });
+      });
+
+      it('creates "optional" environments correctly (hostArgs)', function() {
+        // browsers are irrelevant to this test
+        //
+        if (['firefox', 'chrome', 'remote'].includes(type)) {
+          this.skip();
+          return;
+        }
+
+        let source = '';
+        let hostArguments = '';
+
+        // Setup special cases
+        if (type === 'ch') {
+          hostArguments = '-Intl-';
+          source = 'print(typeof Intl === "undefined");';
+        }
```

Comment:
I've just updated the PR with a new commit that changes ChakraCore to a --debug build of 1.7.0